.travis.yml

```diff
@@ -16,14 +18,8 @@ r_github_packages:
   - jimhester/covr
   
 notifications:
-  email:
```

Comment:
Sorry probably shouldn't remove this, was tinkering with travis to try to get it working but not sure if it helped